frontend/src/app/commonComponents/Button.jsx

```diff
@@ -45,6 +46,7 @@ const Button = ({
       onClick={onClick}
       type={type || "button"}
       disabled={disabled}
+      title={title || label || ""}
```

Comment:
I don't think we want this. We should only add a `title` if the button itself doesn't have a discernible (readable) title, and all of our buttons up to this point have had words. I think it would be better to just pass in a prop that gets spread into the button as `...otherProps`, which would let you put in `title` or `aria-title`.

Reply:
Excellent point. Will remove and let it get passed in via ...otherProps